### ISSUE TITLE
log in issue

### ISSUE THREAD

**Issue by jared00** [A3Wasteland/ArmA3_Wasteland.Altis#362]
I've been trying to play on the A3Wasteland VETERAN/AIRDROPS/5K START/ NEW MISSION map for the last 2 days. It will let me choose team and position and loads mission file but freezes on the load/map screen after that, every time. 
![doesnt work](https://cloud.githubusercontent.com/assets/16126599/11548866/6d663a0a-9926-11e5-8c60-2b834e22f96e.jpeg)

**Comment by hobart666** [comment]
same here not sure whats up with it I can get in IMMEDIATELY after a restart but if I wait a few minutes I cannot and if I do get in right away if I leave I cannot re-enter until another restart...

**Comment by thegoatman** [comment]
If you have 3d markers added remove that from description.ext seems that is blocking the loading.

**Comment by hobart666** [comment]
You sir are the best! now, about this encumbered bs. is there a way to get rid of it's annoyance? I'll check the forums etc. again.

**Comment by dewitjur** [comment]
I'll remove the 3D markers for the CTU Wasteland server. What's your playername Jared? I'm Clouseau

**Comment by thegoatman** [comment]
@hobart666 np

Arma just releases a hotfix seems to fix the running stuff

**Comment by eldrich71** [comment]
do you have to remove or can you just // it out ?

**Comment by dewitjur** [comment]
i removed it awaiting a fix to get 3D back to work without glitching. It worked, loading screen does not freeze anymore

**Comment by hobart666** [comment]
Client side fix it server side? (I'm on the road and can't update it ATM if
server side)

**Comment by thegoatman** [comment]
A client side only fixes it. Am on the road as well my admins reported it worked without a server update ;)

**Comment by thegoatman** [comment]
@eldrich71 you can comment it out didnt test that though.

**Comment by hobart666** [comment]
Perfect! Thanks again, be safe out there.

**Comment by thegoatman** [comment]
@hobart666 same to you sir. FYI See a3wasteland forums posted a fix there for issues with locking vehicles and airdrops if you have them ;)

**Comment by hobart666** [comment]
Thank you I'll check it out I did test apocs airdrops earlier and it was
working but I'll definitely check it all out thoroughly.

**Comment by hobart666** [comment]
@thegoatman has anyone reported anything about the gas stations? Ours aren't working nor are they on the map anymore. Yet they still physically exist on the map.

**Comment by thegoatman** [comment]
@hobart666 yup its reported see http://feedback.arma3.com/view.php?id=26798

**Comment by AgentRev** [comment]
BIS, breaking shit that used to work since 1999.

**Comment by jared00** [comment]
@thegoatman  I have no idea what a 3d marker is or how to take it off lol.

**Comment by thegoatman** [comment]
@jared00 if I read it correct you don't own a server but try to play on one? If thats the case the server owner needs to remove that ;)

**Comment by jared00** [comment]
@thegoatman  that correct. I just tried and it let me on, finallyyyyyy lol. Thats for the tips guys. I gotta admit i'm alittle disappointed with the arma 3 issue reported, its day 3 since I filed an issue with no word back. Thanks for all the comments!

**Comment by jared00** [comment]
now i just realized none of the shop owners are in the locations on the map....

**Comment by thegoatman** [comment]
@jared00 Its caused by some buildings not being recognized as a building anymore see the link above Agentrev's post

**Comment by jared00** [comment]
so we just have to wait till they fix it? another thing, I cant set my helo controls how i want. this "update" screwed everything up...

**Comment by AgentRev** [comment]
lol again? the building issue was present a couple updates back and they fixed it, now it's broken again. fuck you bohemia lol

**Comment by thegoatman** [comment]
@AgentRev yup fuelstations don't even work properly now sometimes they refuel sometimes they don't *sigh* FYI i don't know if the following is caused by something in wasteland or its another BIS mess up but the RPT is showing this:

3:01:33 class HitPoints::HitFuel not found in O_Boat_Armed_01_hmg_F
23:01:33 class HitPoints::HitFuel not found in C_Rubberboat
23:01:33 class HitPoints::HitFuel not found in B_Boat_Transport_01_F

for all vehicles ones

**Comment by AgentRev** [comment]
That's BIS, all code surrounding refueling is set in concrete via the engine and game config.

**Comment by thegoatman** [comment]
@AgentRev thnx reported it to bis http://feedback.arma3.com/view.php?id=26817

**Comment by dewitjur** [comment]
@thegoatman He wanted to play on my server, and the 3d markers have already been removed.

Verzonden met mijn Windows Phone
________________________________
Van: thegoatman<mailto:notifications@github.com>
Verzonden: ‎3/‎12/‎2015 21:53
Aan: A3Wasteland/ArmA3_Wasteland.Altis<mailto:ArmA3_Wasteland.Altis@noreply.github.com>
CC: dewitjur<mailto:dewitjur@hotmail.com>
Onderwerp: Re: [ArmA3_Wasteland.Altis] log in issue (#362)

@jared00 if I read it correct you don't own a server but try to play on one? If thats the case the server owner needs to remove that ;)

---

**Comment by hobart666** [comment]
@thegoatman have you noticed the server rules not being displayed in the list in the map in game? Mine are randomly gone.

**Comment by thegoatman** [comment]
@hobart666 I have the same indeed though its not a game breaking issue so didn't spend any time on it yet to track it down.

**Comment by hobart666** [comment]
Ok making sure it wasn't something I hacked up.
